public/src/app.js

```diff
@@ -76,6 +76,7 @@ let App = {
     ws.on('message', message)
   },
   send(...args) {
+    console.log('Sending', args)
```

Comment:
I know, I bet I won't have time. I'm in the middle of implementing a scientific paper for money, which is really too time-consuming. So that's a throw-away PR for other enthusiasts to come.